packages/navigation-menu/src/lib/navigation-menu/components/sub-menu-item/sub-menu-item.component.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../assets/styles.scss';
+
+li {
+  position: relative;
+
+  a {
+    border-color: $border-color;
+    color: $text-color-light;
+    display: flex;
+    height: $header-height;
+    justify-content: space-between;
+    overflow: hidden;
+    white-space: nowrap;
+
+    &>span.fa {
+      transform: rotate(-90deg);
+    }
+
+    @include at($screen-sm) {
+      // Padding is parent padding + icon size + header padding to the left to align the link to the header text
+      padding-left: calc(#{$menu-docked-width} + #{$submenu-inset});
+      padding-right: $spacer-xs / 2;
+    }
+  }
+
+  ::ng-deep &.a-back-link {
```

Comment:
Please fix indenting

Reply:
actually correct indent, it's included in the parent component